api/test/status.test.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+/**
+ * Test the GET route for home /
+*/
+describe('Dao API', () => {
+    it('returns status to be OK', async () => {
+        const apiUrl = 'http://localhost:2015/';
```

Comment:
You can create a file called utils.js and have the apiurl there. and import it and use it here. So, you dont have to copy paste it every time and if the API url changes we change it in one place.

Reply:
Noted sir.

Reply:
Done